src/controls/NumberControl.test.tsx

```diff
@@ -118,17 +116,6 @@ it ("renders slider and input box when number of steps is greater than threshold
   expect(screen.getByRole("spinbutton"))
 })
 
-it ("renders slider without input box when number of steps is less than threshold", () => {
-  const data = { basisPoints: 1 }
-  render({
-    data: data,
-    schema: numberHumiditySchema,  // 100 steps
-    uischema: numberPercentageUISchema,
-  })
-  expect(screen.getByRole("slider"))
-  expect(screen.queryByRole("spinbutton")).toBeNull()
-})
```

Comment:
Test no longer applies since we aren't conditionally adding the text input with the slider anymore